server/conditions/space_collision_condition.go

```diff
@@ -0,0 +1,68 @@
+package conditions
+
+import (
+	"fmt"
+	"github.com/bitdecaygames/fireport/server/actions"
+	"github.com/bitdecaygames/fireport/server/pogo"
+)
+
+// SpaceCollisionCondition address the case of two players moving into the same space
+type SpaceCollisionCondition struct{}
```

Comment:
Here is the condition that checks for two players occupying the same space.  I noticed some overlap in how this one and the `EdgeCollisionCondition` did things.  So this might be ripe for a refactor to appease DRY.